src/common/ImageAvatar.js

```diff
@@ -11,7 +12,22 @@ const styles = StyleSheet.create({
   },
 });
 
-export default({ avatarUrl, size, status, isCircular, onPress }) => {
+type funcArguments = {
+  avatarUrl: string,
+  size: number,
+  status?: string,
+  isCircular?: boolean,
+  onPress?: () => void
+}
+
+
+export default({
+                 avatarUrl,
```

Comment:
Can we adjust the format of this block to something like:
```
export default({
  avatarUrl,
  size,
  ...,
}: funcArguments) => {
  ...
}
```
It's just that it's more consistent with the rest of the code :)